aws/codecommit/module.tf

```diff
@@ -0,0 +1,130 @@
+terraform {
+  required_version = "~> 0.11.14"
+
+  # The configuration for this backend will be filled in by Terragrunt
+  backend "s3" {}
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Providers
+# ----------------------------------------------------------------------------------------------------------------------
+
+provider "aws" {
+  version = "~> 2.11.0"
+  region  = "${var.aws_region}"
+
+  assume_role {
+    role_arn = "arn:aws:iam::${var.account_id}:role/${var.account_assume_role_name}"
+  }
+}
+
+provider "aws" {
+  alias   = "master"
+  version = "~> 2.11.0"
+  region  = "${var.aws_region}"
+
+  assume_role {
+    role_arn = "arn:aws:iam::${var.master_account_id}:role/${var.master_account_assume_role_name}"
+  }
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Module Shared Variables
+# ----------------------------------------------------------------------------------------------------------------------
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Module Standard Variables
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "name" {
+  default = "codecommit"
+}
+
+variable "create" {
+  default = "true"
+}
+
+variable "aws_region" {
+  default = "us-east-1"
+}
+
+variable terraform_module {
+  default = "gravicore/terraform-gravicore-modules/aws/codecommit"
+}
+
+# ----------------------------------------------------------------------------------------------------------------------
+# Platform Standard Variables
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "namespace" {
+  default = "grv"
+}
+
+variable "environment" {}
+
+variable "stage" {
+  default = "dev"
+}
+
+variable "repository" {
+  default = ""
+}
+
+variable "master_account_id" {}
+variable "account_id" {}
+
+variable "master_account_assume_role_name" {
+  default = "grv_deploy_svc"
```

Comment:
Do you want to start switching to `grv-deployment-service` from `grv_deploy_svc`? or is this a non issue with overrides?